Source/Core/Core/HW/GCKeyboardEmu.cpp

```diff
@@ -248,27 +248,27 @@ void GCKeyboard::LoadDefaults(const ControllerInterface& ciface)
   m_keys4x->SetControlExpression(4, ";");
   m_keys4x->SetControlExpression(5, "Backspace");
   m_keys4x->SetControlExpression(6, "Tab");
-  m_keys4x->SetControlExpression(7, "Caps Lock");
-  m_keys4x->SetControlExpression(8, "Left Shift");
-  m_keys4x->SetControlExpression(9, "Right Shift");
-  m_keys4x->SetControlExpression(10, "Left Control");
-  m_keys4x->SetControlExpression(11, "Right Alt");
-  m_keys4x->SetControlExpression(12, "Left Command");
+  m_keys4x->SetControlExpression(7, "`Caps Lock`");
+  m_keys4x->SetControlExpression(8, "`Left Shift`");
+  m_keys4x->SetControlExpression(9, "`Right Shift`");
+  m_keys4x->SetControlExpression(10, "`Left Control`");
+  m_keys4x->SetControlExpression(11, "`Right Alt`");
+  m_keys4x->SetControlExpression(12, "`Left Command`");
   m_keys4x->SetControlExpression(13, "Space");
-  m_keys4x->SetControlExpression(14, "Right Command");
+  m_keys4x->SetControlExpression(14, "`Right Command`");
   m_keys4x->SetControlExpression(15, "");  // Menu
 
-  m_keys5x->SetControlExpression(0, "Left Arrow");
-  m_keys5x->SetControlExpression(1, "Down Arrow");
+  m_keys5x->SetControlExpression(0, "`Left Arrow`");
+  m_keys5x->SetControlExpression(1, "`Down Arrow`");
   m_keys5x->SetControlExpression(2, "Up Arrow");
```

Comment:
Did you leave that one un-ticked intentionally?

Reply:
No, I just missed it :) 👍